scalameta/parsers/js/npm/README.md

```diff
@@ -0,0 +1,9 @@
+# scalameta-parsers
+
+A JS API for the `parsers` module of [`scalameta`](https://github.com/scalameta/scalameta).
+
+It exposes two functions:
```

Comment:
Since we're exporting a commonjs module, it needs a `require`, I'll add an example.